src/App.test.js

```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import React from 'react';
+import { render } from '@testing-library/react';
+import Navbar from './components/navbar/Navbar';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+test('Navbar component renders without errors', () => {
+  const { container } = render(<Navbar darkMode toggleDarkMode={() => {}} />);
```

Comment:
- [OPTIONAL] You have to improve your design styling to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)) including:
* Typography: font face, size, and weight
* Layout: composition and space between elements to follow the design guidelines including:
  - Typography: font face, size, and weight
  - Layout: composition and space between elements especially for the `stockDetail` page. 👇🏻 
 
![image](https://github.com/DevKaranJ/stock-watch/assets/79058364/2e7b75d3-2732-460f-96bb-d7d433be66c5)

- Your Design 👇🏻 

![image](https://github.com/DevKaranJ/stock-watch/assets/79058364/e657900c-c89a-46ad-9a94-6f4974c85a19)